src/schema/objects/datatypes.yaml

```
@@ -16,14 +16,15 @@ dwi:
   display_name: Diffusion-Weighted Imaging
   description: |
     Diffusion-weighted imaging (DWI).
+ecephys:
+  value: ecephys
+  display_name: Extracellular Electrophysiology
+  description: |
+    Electrophysiological data acquired from an extracellular space
```

@bids-standard/bep032 is this the best way to define `ecephys`?

---

I'd say “electrophysiological data from extracellular microelectrodes”

---

ok, let's do that then

```suggestion
    Electrophysiological data from an extracellular microelectrodes
```